sdk/identity/Azure.Identity/src/Credentials/InteractiveBrowserCredential.cs

```diff
@@ -21,6 +21,9 @@ public class InteractiveBrowserCredential : TokenCredential
         internal string[] AdditionallyAllowedTenantIds { get; }
         internal string ClientId { get; }
         internal string LoginHint { get; }
+        internal bool? UseEmbeddedWebView { get; }
+        internal SystemWebViewOptions SystemBrowserOptions { get; }
+        internal EmbeddedWebViewOptions EmbeddedBrowserOptions {get;}
```

Comment:
Why not clone the whole `BrowerCustomizationOptions` instead of adding 3 distinct properties?